src/openloco/windows/TimePanel.cpp

```diff
@@ -330,39 +331,40 @@ namespace openloco::ui::TimePanel
     // 0x0043995C
     void formatChallenge()
     {
-        _common_format_args[0] = current_day();
+        FormatArguments args = {};
+        args.push(current_day());
 
         auto playerCompany = companymgr::get(companymgr::get_controlling_id());
 
         if ((playerCompany->challenge_flags & company_flags::challenge_completed) != 0)
         {
-            _common_format_args[2] = string_ids::challenge_completed;
+            args.push(string_ids::challenge_completed);
         }
         else if ((playerCompany->challenge_flags & company_flags::challenge_failed) != 0)
         {
-            _common_format_args[2] = string_ids::challenge_failed;
+            args.push(string_ids::challenge_failed);
         }
         else if ((playerCompany->challenge_flags & company_flags::challenge_beaten_by_opponent) != 0)
         {
-            _common_format_args[2] = string_ids::empty;
+            args.push(string_ids::empty);
         }
         else
         {
-            _common_format_args[2] = string_ids::challenge_progress;
-            _common_format_args[3] = playerCompany->var_8C4E;
+            args.push(string_ids::challenge_progress);
+            args.push(playerCompany->var_8C4E);
```

Comment:
This is an 8bit variable. It needs to be 16bit

Reply:
Okay, so we can invoke:
```suggestion
            args.push<uint16_t>(playerCompany->var_8C4E);
```